content/applications/hr/fleet/service.rst

```diff
@@ -6,72 +6,282 @@ To properly maintain a fleet of vehicles, regular maintenance as well as periodi
-  button next to the vehicle selected in the :guilabel:`Vehicle` field. Change the unit of measure,
-  then click :guilabel:`Save`. the unit of measure will be updated in the :guilabel:`Odometer Value`
-  field.
+  measure will either be in kilometers (:guilabel:`km`) or miles (:guilabel:`mi`), depending on how
+  the selected vehicle was configured.
+
+  When the :guilabel:`Vehicle` is selected, the units of measure is populated. This comes from the
+  vehicle form.
+
+  To change from kilometers to miles, or vice versa, click the :guilabel:`Internal Link` button to
+  the right of the vehicle selected in the :guilabel:`Vehicle` field.
+
+  Change the unit of measure, then navigate back to the service form via the
+  :guilabel:`breadcrumbs`. The unit of measure is updated in the :guilabel:`Odometer Value` field.
 - :guilabel:`Notes`: enter any notes for the repair in the notes tab at the bottom of the service
   form.
 
 .. image:: service/new-service.png
    :align: center
    :alt: Enter the information for a new service. The required fields are Service Type and Vehicle.
 
+.. _service/new-type:
+
+Create service type
+-------------------
+
+Service types must be created from a service form, there is no other way to access the list of
+service types.
+
+On the :ref:`service form <service/service-form>`, type in the name of the new :guilabel:`Service
+Type` in the field. Then, click :guilabel:`Create and edit...` and a :guilabel:`Create Service Type`
+form appears.
+
+The service type entered on the service form populates the :guilabel:`Name` field by default. This
+field can be modified if desired.
+
+Using the drop-down menu, select the :guilabel:`Category` this new service type falls under, if any.
+The two default options to chose form are :guilabel:`Contract` or :guilabel:`Service`. Additional
+categories cannot be created.
+
+If the service applies to **only** contracts or services, select the corresponding
+:guilabel:`Category`. If the service applies to **both** contracts *and* services, leave this field
+blank.
+
+When done, click :guilabel:`Save & Close`.
+
+.. _service/new-vendor:
+
+Create vendor
+-------------
+
+When a service is done for the first time, typically the vendor is not already in the database. It
+is best practice to add the full details for a vendor in the database, so any necessary information
+is easily retrieved.
+
+On the :ref:`service form <service/service-form>`, type in the name of the new :guilabel:`Vendor`
+in the field. Then, click :guilabel:`Create and edit...` and a :guilabel:`Create Vendor` form
+appears.
+
+The vendor name entered on the service form populates the :guilabel:`Name` field by default. This
+field can be modified if desired.
+
+.. note::
+   Depending on what other applications are installed, different tabs or fields may be visible on
+   the :guilabel:`Create Vendor` form.
+
+General information
+~~~~~~~~~~~~~~~~~~~
+
+Fill out the following information on the top-half of the form:
```

Comment:
This section is correct and I'm sure whoever's reading it would appreciate the detail, but I'm not really sure if the information is much different from filling out a contact form. Do you have existing documentation on how to fill out a contact form that you can link to? 

In this section, I'd personally just include information specific and unique to filing out vendor forms, and shorten this section a little more. However, this is a nitpick.

I think this section is good to publish as-is for this PR, though. Personally, I'd shorten this section and link to the doc to create a new contact form in a follow-up PR, and boost your KPIs (; However, you can totally just disregard this comment entirely haha

Reply:
I do not- but I do reference a very similar form in another doc but it is for a totally different kind of form (and I am trying to remember where it was!). I would link IF there was a doc like this, but I would need to make one and figure out where it goes- as in, how often is this kind of form found and in which apps. I think that is a good idea for the future- but for now, I like that everything they need is in here. I do think that it is a good idea to make a separate doc for this- and link that to the places it would be appropriate- but since I don't know where that would be right now (and the information is necessary for this doc as-is/since there is no doc like that yet that I found) I will keep it for this version/revision.